src/v3/src/util/theme.ts

```diff
@@ -294,6 +294,7 @@ export const createTheme = (
                 borderColor: 'transparent',
               },
             }),
+            textTransform: 'none',
```

Comment:
button text was being capitalized when running both providers

Reply:
This is a little concerning - I think the 1.x theme provider is supposed to be totally scoped to its own components. Can you look into why this is happening before merging?

Reply:
so far ive pin pointed the issue to [this](https://github.com/okta/okta-signin-widget/blob/7d813b440e24dff415eb2cfe19d8f751e46641d7/src/v3/src/util/theme.ts#L176) line where we are overriding the button font family.  Since we are overriding the theme in the new theme provider, overriding the button font family causes the button typography theme to merge with the default mui button typography theme when the theme is created [here](https://github.com/okta/odyssey/blob/34a3299b031dd93be1b13ba3c9b69fa929311d04/packages/odyssey-react-mui/src/OdysseyThemeProvider.tsx#L52).  
Looks like this is where the uppercase transform is added: https://github.com/mui/material-ui/blob/42f78c69a59f0b5fa5fb8f533385f04ef77d7889/packages/material-ui/src/styles/createTypography.js#L68
and
https://github.com/mui/material-ui/blob/42f78c69a59f0b5fa5fb8f533385f04ef77d7889/packages/material-ui/src/styles/createTypography.js#L8C1-L8C1

Reply:
bug ticket filed to odyssey and comment added